packages/client/lib/Client.js

```diff
@@ -148,6 +152,14 @@ export default class Client {
   get collateral () {
     return this._collateral
   }
+
+  get loan () {
```

Comment:
what's the difference with the `collateral` side? It seems we are expanding the namespaces quite quickly for your usecase. Perhaps it could all come under `loan` and then you can have sub namespaces like `loan.fund.xxxxxx``loan.collateral.xxxxxx` ?